src/products/voice/utils/actionBuilder.ts

```diff
@@ -0,0 +1,204 @@
+import joi from 'joi';
+import {
+  Action, Child, SayAttributes, SayOrPlayChildren, GetDigitsAttributes,
+  GetDigitsCombined, DialAttributes, RecordAttributes, EnqueueAttributes,
+  DequeueAttributes, RedirectAttributes,
+} from './actionBuilder.types';
+import { validateJoiSchema } from '../../../utils/misc';
+
+export class ActionBuilder {
```

Comment:
This action builder is great 🔥 

We need to cater for nesting of actions though, to reflect a more realistic usage of the voice API e.g

```javascript
const response = '<Response>' +
                    '<GetDigits timeout="8">' +
                    '<Play url="http://source.com/some_file.mp3"/>' +
                    '</GetDigits>' +
                    '</Response>';
```

Reply:
Hey @tawn33y never mind this, I just noticed that you've implemented this as well. 🙌